webapi/src/main/java/com/dumbdogdiner/stickyapi/util/http/HttpConnectionException.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2020-2021 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.util.http;
+
+import okhttp3.HttpUrl;
+import okhttp3.Request;
+
+import java.io.IOException;
+import java.net.URL;
+
+/**
+ * Runetime exception to wrap IOExceptions from HTTP requests
+ */
+public class HttpConnectionException extends HttpException {
```

Comment:
Is this entirely necessary? Especially since you're already wrapping your own HttpException class

Reply:
I shouldn't be wrapping it, it should be extending. If I'm wrapping it that will be removed